living_documentation_regime/exporter/mdoc_exporter.py

```diff
@@ -80,117 +97,205 @@ def export(self, **kwargs) -> bool:
         return True
 
     def _generate_report_page(self):
-        header, divider, *error_rows = self._report_page_content.strip().split("\n")
-        if error_rows:
-            report_page_content = "\n".join([header, divider] + error_rows)
-            report_page = self._report_page_template.format(
-                date=datetime.now().strftime("%Y-%m-%d"), livdoc_report_page_content=report_page_content
-            )
-            with open(
-                os.path.join(make_absolute_path(self._output_path), "report_page.md"), "w", encoding="utf-8"
-            ) as f:
-                f.write(report_page)
+        def write_report_page(group: str, parent_dir: str, content: str) -> None:
+            header, divider, *error_rows = content.strip().split("\n")
+            if error_rows:
+                report_page_content = "\n".join([header, divider] + error_rows)
```

Comment:
you split it to join it again? Why not just use the original content.

Reply:
Fixed in commit - 7fca425e94810142c482e2a077968e4f5c1e7ec2.